zeno/classes/classes.py

```diff
@@ -48,9 +49,15 @@ class TableRequest(CamelModel):
     slice_range: List[int]
     filter_predicates: FilterPredicateGroup
     sort: Tuple[Union[ZenoColumn, None], bool]
+    tag_ids: FilterIds
     filter_ids: Optional[FilterIds] = None
 
 
+class PlotRequest(CamelModel):
```

Comment:
What is this used for? IDK if I like the name too much. Any other options?

Reply:
This is used to query all the IDs that have been selected by filter predicates and tags for the scatterplot (getFilteredIds function), which is why I named it PlotRequest to mimic the TableRequest naming.